js/app/interfaces/documentCard.js

```diff
@@ -71,4 +71,15 @@ const DocumentCard = new Lang.Interface({
      *
      */
     clear_content: Lang.Interface.UNIMPLEMENTED,
+
+    /**
+     * Method: stop_content
+     * Stop the content for this article. Different from
+     * clear in the sense that we don't want to destroy
+     * the content itself, since it may be used again if user
+     * returns to article page. But for e.g. video and audio
+     * content we want to stop it playing.
+     *
+     */
+    stop_content: Lang.Interface.UNIMPLEMENTED,
```

Comment:
I am not really good at naming things but is there any way to make this method more generic?
So that it could eventually used for other things

maybe set_active (is_active)